clients/drcachesim/tests/warmup-zeros.templatex

```diff
@@ -3,14 +3,14 @@ Hello, world!
 Cache simulation results:
 Core #0 \(1 thread\(s\)\)
   L1I stats:
-    Warmup hits:                  *[0-9,\.]*..
-    Warmup misses:                *[0-9,\.]*..
+    Warmup hits:                  *[0]*..
```

Comment:
OK, I expected 0 with nothing after it (the dots on the others are matching numbers at least N digits long -- I suppose they should be digits and not dots, that's just the way the first one was written and then we copied from there...).